src/index.ts

```diff
@@ -86,6 +96,32 @@ const setupServer = async () => {
 
         const result = await addUser(userData);
         return result;
+      },
+      async login(_, args) {
+        console.log(args.data);
+        const user = await AppDataSource.manager.findOneBy(User, {
+          email: args.data.email
+        });
+        console.log(user);
+        if (!user) {
+          throw new CustomError('Unable to login', 404);
+        }
+        const isMatch = await bcrypt.compare(args.data.password, user.password);
```

Comment:
```suggestion
        const isPasswordVaid = await bcrypt.compare(args.data.password, user.password);
```

Talvez só pra especificar melhor que se trata de password.